test_crates/declarative_macro_missing/old/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+#[macro_export]
+macro_rules! will_be_removed {
+    () => {};
+}
+
+#[macro_export]
+macro_rules! will_no_longer_be_exported {
+    () => {};
+}
+
+macro_rules! textual_scope_macro_removed {
+    () => {};
+}
```

Comment:
A couple more test cases here.
```suggestion
}

#[macro_export]
macro_rules! became_doc_hidden {
    () => {};
}

#[doc(hidden)]
#[macro_export]
macro_rules! always_doc_hidden {
    () => {};
}

```